src/capturer/engine/win/mod.rs

```diff
@@ -21,17 +20,6 @@ struct Capturer {
     pub crop: Option<Area>,
 }
 
-impl Capturer {
```

Comment:
Has this change been tested on windows? From what I remember, this was required to make it work on windows.

Reply:
Yup. I coded the windows part on windows ✅